content/news/033/index.md

```diff
@@ -179,6 +179,18 @@ _Discussions:
 
 ## Learning Material Updates
 
+### [Why I choose to build my game from scratch][why-article]
+
+[@HeavyRain266] published an article [Why I choose to build my game from scratch][why-article],
+short story about implementation of the game "Forbidden Valley" from scratch in Rust.
+Author tries to point out, how much you can learn from building your dream game
+without help of any game engine.
+
+_Discussions: [r/rust_gamedev][why-article]_
+
+[@HeavyRain266]: https://github.com/HeavyRain266
+[why-article]: https;//reddit.com
```

Comment:
bad link

Reply:
Oh woops, I forgot to replace that placeholder.

Reply:
Fixed.